src/app/events/[slug]/components/SponsorContainer.tsx

```diff
@@ -3,21 +3,26 @@ import { clsx } from 'clsx'
 import { type SponsorLogoProps, SponsorLogo } from './SponsorLogo'
 import type { Sponsor } from './SponsorSection'
 
-type SponsorGridProps = {
+type SponsorContainerProps = {
   sponsors: Array<Sponsor>
   tier: string
-  gridClassName: string
   logoImageConfig: SponsorLogoProps['logoImageConfig']
+  gap?: string
 }
 
-export function SponsorGrid({
+export function SponsorContainer({
   sponsors,
   tier,
-  gridClassName,
   logoImageConfig,
-}: SponsorGridProps) {
+  gap = 'gap-8',
+}: SponsorContainerProps) {
   return (
-    <div className={clsx('grid gap-8', gridClassName)}>
+    <div
+      className={clsx(
+        gap,
+        'flex flex-col flex-wrap justify-start sm:flex-row sm:justify-center sm:gap-8',
+      )}
+    >
```

Comment:
Charly suggested the other day to define the known styles first .. I think it makes sense!

```suggestion
    <div
      className={clsx(
        'flex flex-col flex-wrap justify-start sm:flex-row sm:justify-center sm:gap-8',
        'gap'
      )}
    >
```

Reply:
Also - Shall we move `sm:gap-8` to be part of the default?